esy-lib/System.ml

```diff
@@ -107,4 +107,6 @@ module Environment = struct
     | Some path -> String.split_on_char sep.[0] path
     | None -> []
 
+  let normalizeNewLines s =
```

Comment:
Wasn't sure the best spot for this - let me know if you have an idea for a better place!

Reply:
That's fine!